sdk/app_cpu1/user/usr/user_config.h

```diff
@@ -13,6 +13,10 @@
 // System uses 10kHz by default
 //#define SYS_TICK_FREQ (20000) // Hz
 
+// Configure the ratio between the carrier frequency to the control frequency
+// System uses 20Hz by default
+#define SYS_PWM_CARRIER_CONTROL_RATIO (20)
+
```

Comment:
Update this to better explain what this number does. Give an example. Should not be "20Hz"

Reply:
I think I included this at the very beginning when I was using what was already in #318 - I'm going to go ahead and remove it since its not being used